.storybook/config.js

```diff
@@ -58,7 +58,7 @@ configure(() => {
   require('./../packages/bpk-component-chip/stories');
   require('./../packages/bpk-component-close-button/stories');
   require('./../packages/bpk-component-code/stories');
-  require('./../packages/bpk-component-definition-list/stories');
+  require('./../packages/bpk-component-description-list/stories');
```

Comment:
Just needs to be one line lower, as right now it's appearing before `bpk-component-content-container`.

Reply:
Good catch - should actually be 2 lines lower 😉  😂

Reply:
Close enough :P